pkgs/applications/window-managers/hyprwm/hyprland/default.nix

```diff
@@ -53,41 +56,46 @@ assert lib.assertMsg (!hidpiXWayland) "The option `hidpiXWayland` has been remov
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "hyprland" + lib.optionalString debug "-debug";
-  version = "0.40.0-unstable-2024-05-05";
+  version = "0.41.0";
 
   src = fetchFromGitHub {
     owner = "hyprwm";
     repo = finalAttrs.pname;
     fetchSubmodules = true;
-    rev = "f15513309b24790099d42974274eb23f66f7c985";
-    hash = "sha256-zKOfgXPTlRqCR+EME4qjN9rgAnC3viI5KWx10dhKszw=";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-iX/l3UT8iXu8psu2UirFX11Yg2zYwpgzoXB32oM3N3U=";
   };
 
   postPatch = ''
     # Fix hardcoded paths to /usr installation
     sed -i "s#/usr#$out#" src/render/OpenGL.cpp
+
+    # Remove extra @PREFIX@ to fix pkg-config paths
+    sed -i "s#@PREFIX@/##g" hyprland.pc.in
   '';
 
   # used by version.sh
-  DATE = "2024-05-05";
```

Comment:
Why is this removed? It's still being picked up by `version.sh`.

Reply:
I thought that was for just unstable git Hyprland?

Is there a way to do automate it, perhaps on the source side, a file to fetch which contains the date?

Reply:
Technically git should provide the date, as it does when using flakes. But I don't see it being provided by `fetchgit`.

Reply:
I think that requires `leaveDotGit = true;` but it turns the output nondeterministic.

Reply:
Another thing, we can't just use this anymore, since we are moving to a tagged release.
https://github.com/NixOS/nixpkgs/blob/07c3ec6a8975b940e96f1db59d5650a3b25bf73a/pkgs/applications/window-managers/hyprwm/hyprland/default.nix#L78

Reply:
No, but you get the `.git` that could be used to extract the date. But again it turns the output nondeterministic as .git will change with upstream 🤔

Reply:
Could this be used? 
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/set-source-date-epoch-to-latest.sh

Reply:
> Another thing, we can't just use this anymore, since we are moving to a tagged release.
> 
> https://github.com/NixOS/nixpkgs/blob/07c3ec6a8975b940e96f1db59d5650a3b25bf73a/pkgs/applications/window-managers/hyprwm/hyprland/default.nix#L78

Right, let's drop it then. We can drop the date as well, if we only build tagged releases.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/319131/commits/f6b173654a0b22be3087e97d622f3b46603f92fd